src/LibraryUtilities.ts

```diff
@@ -747,7 +747,19 @@ export function getHighlightedText(text: string, highlightedText: string, matchD
  * @return {boolean} true if an item is found, false otherwise
  */
 export function findAndExpandItemByPath(pathToItem: ItemData[], allItems: ItemData[]): boolean {
-    let item: ItemData;
+
+    //If RawDataType is handcrafted, then there is a chance for the item
+    //to get incorrect information. Otherwise, pathToItem has the right members
+    //just expanding the item to true will work. 
+    pathToItem.forEach(element => {
+        element.expanded = true;
+    });
+
+    return true;
+   
+    //if the above code did not work, then uncomment the below
+    //code to find the actual issue.
+   /* let item: ItemData;
```

Comment:
Did you find out any case you will need to uncomment this below yet?

Reply:
not yet. still testing